PermanentOrg/web-app
# PER-8919: Public Archives Landing Page

Steps to test:
1. Go to the app gallery tab from the left side nav bar.
2. A list of public archives should be displayed, or a message if no public archives exist.

- Added public archives component in which a list of public archives is displayed, with the option to click on an archive and getting redirected to the archive page.
- Added access role pipe to display the access role of the user in the public archives list.
- Added unit tests for the new pipe and the new component.
